setup.py

```diff
@@ -59,10 +59,9 @@
                         'katsdptelstate[rdb]', 'dask[array]'],
       extras_require={
           'ms': ['python-casacore >= 2.2.1', 'numba'],
-          's3': ['requests', 'defusedxml'],
           # rados is not in PyPI but available as Debian package python-rados
           'rados': ['rados'],
-          # katsdpauth is currently only available via GitHub
-          'auth': ['katsdpauth'],
+          's3': ['requests', 'defusedxml', 'oic'],
+          'auth': [],
```

Comment:
It is OK to remove this now.

Reply:
Oh good, I did that in the other branch now.